hal/targets.json

```diff
@@ -2083,20 +1988,29 @@
         "extra_labels": ["STM", "STM32F1", "STM32F103C8"],
         "supported_toolchains": ["GCC_ARM"],
         "inherits": ["Target"],
-        "progen": {"target": "bluepill-f103c8"},
         "device_has": ["ANALOGIN", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "SERIAL", "SLEEP", "SPI", "SPISLAVE"]
     },
     "NUMAKER_PFM_NUC472": {
         "core": "Cortex-M4F",
         "default_toolchain": "ARM",
         "extra_labels": ["NUVOTON", "NUC472", "NUMAKER_PFM_NUC472"],
-		"macros": ["MBEDTLS_ENTROPY_HARDWARE_ALT"],
+        "macros": ["MBEDTLS_ENTROPY_HARDWARE_ALT"],
         "is_disk_virtual": true,
         "supported_toolchains": ["ARM", "uARM", "GCC_ARM", "IAR"],
         "inherits": ["Target"],
-        "progen": {"target": "numaker-pfm-nuc472"},
         "device_has": ["ANALOGIN", "I2C", "I2CSLAVE", "I2C_ASYNCH", "INTERRUPTIN", "LOWPOWERTIMER", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SERIAL_ASYNCH", "SERIAL_FC", "SLEEP", "SPI", "SPISLAVE", "SPI_ASYNCH"],
         "features": ["IPV4"],
+        "release_versions": ["2", "5"],
+        "device_name": "NUC472HI8AE"
+    },
+    "NCS36510": {
```

Comment:
It looks like there was an issue rebasing, There are two entries of NCS36510 now.